container/public/LuigiContainer.svelte.d.ts

```diff
@@ -19,6 +19,16 @@ export default class LuigiContainer extends HTMLElement {
    */
   webcomponent: boolean;
 
+  /**
+   * If set to `true` a microfrontend will be immediately initialized. LuigiContainer sends an event `initialized` to the microfrontend.
+   */
+  can_mfe_initialized: boolean;
+
+  /**
+   * Will be set to `true` if `can_mfe_initialized` is set to `true` otherwise it will be `true` if the mfe webcomponent is attached to the container.
```

Comment:
I find it a bit misleading. 

> **Will be set to true** if can_mfe_initialized is set to true **otherwise it will be `true`** if the mfe webcomponent is attached to the container.

        (can_mfe_initialized = true) => (mfe_initialized_ready = true)  then also: 
         <=> (can_mfe_initialized = false) => (mfe_initialized_ready = false) so aren't 
         can_mfe_initialized = mfe_initialized_ready so they are the same, then why use two?
         
Or maybe the wording is just not right.

Reply:
ok.. maybe I need other names :)